lib/src/main/java/graphql/nadel/validation/NadelSchemaValidationError.kt

```diff
@@ -409,6 +409,20 @@ sealed interface NadelSchemaValidationError : NadelSchemaValidationResult {
 
         override val subject = overallField
     }
+
+    data class AllFieldsUsingHiddenDirective(
+        val parentType: NadelServiceSchemaElement,
+    ) : NadelSchemaValidationError {
+        val service: Service get() = parentType.service
+
+        override val message = run {
+            val s = service.name
+            val ut = parentType.underlying.name
+            "No fields found without @hidden directive on the underlying type $ut on service $s"
```

Comment:
`@hidden` is applied on the overall type and so we don't need to share the `service`

So this can be something like

```kotlin
val ot = parentType.overall.name
"Must have at least one field without @hidden on type $ot"
```

Reply:
Got it, done!